packages/abstractions/src/multipartBody.ts

```diff
@@ -128,13 +131,18 @@ export function serializeMultipartBody(
     writer.writeStringValue(undefined, "--" + boundary);
     const part = parts[partName];
     writer.writeStringValue("Content-Type", part.contentType);
+    let dispositionValue = 'form-data; name="' + part.originalName + '"';
```

Comment:
can you add additional tests here https://github.com/microsoft/kiota-typescript/blob/main/packages/serialization/multipart/test/common/multipartSerializationWriter.ts for that additional behaviour please?

Reply:
addOrReplacePart changes are not consistent across languages, isn't it a problem?

Reply:
https://www.ietf.org/rfc/rfc2388.txt

it seems that filename is not mandatory.

> The original local file name may be supplied as well, either as a
>    "filename" parameter either of the "content-disposition: form-data"
>    header or, in the case of multiple files, in a "content-disposition:
>    file" header of the subpart. The sending application MAY supply a
>    file name; if the file name of the sender's operating system is not
>    in US-ASCII, the file name might be approximated, or encoded using
>    the method of RFC 2231.

Reply:
> addOrReplacePart changes are not consistent across languages, isn't it a problem?

Can you provide more context for this please?